src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -359,7 +359,8 @@ void minimize_local_reorders(program& p, std::map<program_node*, format::type>&
             auto reorders_cnt = count_reorders(fmt_map, lo, node);
 
             if (reorders_cnt.number < best_reorder_cnt.number ||
-                (reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes < best_reorder_cnt.total_sizes) ) {
+                (reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes < best_reorder_cnt.total_sizes) ||
+                (reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes == best_reorder_cnt.total_sizes)) {
```

Comment:
why not just merge those two conditions? 
(reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes <= best_reorder_cnt.total_sizes)

And did you check the overall performance impact?

Reply:
Yes, it can be merged. And only check the target network, I also agree PR will be merged after overall performance test.